sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/operation/ExecuteStatementOperation.java

```diff
@@ -67,4 +68,16 @@ protected void setConfOverlay(Map<String, String> confOverlay) {
       this.confOverlay = confOverlay;
     }
   }
+
+  protected void registerCurrentOperationLog() {
+    if (isOperationLogEnabled) {
+      if (operationLog == null) {
+        LOG.warn("Failed to get current OperationLog object of Operation: " +
+                getHandle().getHandleIdentifier());
```

Comment:
I think the indentation should be double spaced.

Reply:
This was resolved by me when I did the merge.

Reply:
Thank you @gatorsmile.

Reply:
thanks @gatorsmile